src/test/java/com/binance/dex/api/client/examples/HtltExample.java

```diff
@@ -0,0 +1,55 @@
+package com.binance.dex.api.client.examples;
+
+import com.binance.dex.api.client.BinanceDexApiClientFactory;
+import com.binance.dex.api.client.BinanceDexApiRestClient;
+import com.binance.dex.api.client.BinanceDexEnvironment;
+import com.binance.dex.api.client.Wallet;
+import com.binance.dex.api.client.domain.TransactionMetadata;
+import com.binance.dex.api.client.domain.broadcast.HtltReq;
+import com.binance.dex.api.client.domain.broadcast.TransactionOption;
+import com.binance.dex.api.client.encoding.EncodeUtils;
+import com.binance.dex.api.client.encoding.message.Token;
+import org.apache.commons.lang3.ArrayUtils;
+import org.bouncycastle.util.encoders.Hex;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author: fletcher.fan
+ * @create: 2019-10-09
+ */
+public class HtltExample {
+
+    public static void main(String[] args) throws IOException, NoSuchAlgorithmException {
+        String yourPrivateKey = "";
+        Wallet wallet =
+                new Wallet(yourPrivateKey,
+                        BinanceDexEnvironment.TEST_NET);
+        BinanceDexApiRestClient client =
+                BinanceDexApiClientFactory.newInstance().newRestClient(BinanceDexEnvironment.TEST_NET.getBaseUrl());
+
+        HtltReq htltReq = new HtltReq();
+        htltReq.setRecipient(""); // the recipient
+        htltReq.setRecipientOtherChain("");
+        htltReq.setSenderOtherChain("");
+        long timestamp = Instant.now().getEpochSecond();
+        byte[] originData = ArrayUtils.addAll(Hex.decode(wallet.getPrivateKey()), EncodeUtils.long2Bytes(timestamp));
```

Comment:
Why not just random generate a 32-length byte array as the random number? It is wierd to use wallet key as random number?